BossMod/BossModule/AIHints.cs

```diff
@@ -10,7 +10,7 @@ public class Enemy(Actor actor, bool shouldBeTanked)
         public const int PriorityForbidFully = -2; // attacking this enemy is forbidden both by ai or player (e.g. invulnerable, or attacking/killing might lead to a wipe)
 
         public Actor Actor = actor;
-        public int Priority = actor.InCombat || actor.FateID != 0 ? 0 : PriorityForbidAI; // <0 means damaging is actually forbidden, 0 is default (TODO: revise default...)
+        public int Priority = PriorityForbidAI; // <0 means damaging is actually forbidden, 0 is default (TODO: revise default...)
```

Comment:
There are a bunch of Enemy instances created outside of FillPotentialTargets (i.e. arr titan modules), they probably need to be updated...

Reply:
no i can just revert this, since i'm always setting priority in FillPotentialTargets anyway